rabbitmq-operator/src/main/java/com/indeed/operators/rabbitmq/reconciliation/rabbitmq/OperatorPolicyReconciler.java

```diff
@@ -0,0 +1,81 @@
+package com.indeed.operators.rabbitmq.reconciliation.rabbitmq;
+
+import com.indeed.operators.rabbitmq.api.RabbitManagementApiProvider;
+import com.indeed.operators.rabbitmq.model.crd.rabbitmq.OperatorPolicySpec;
+import com.indeed.operators.rabbitmq.model.rabbitmq.RabbitMQCluster;
+import com.indeed.operators.rabbitmq.model.rabbitmq.RabbitMQConnectionInfo;
+import com.indeed.operators.rabbitmq.resources.RabbitMQServices;
+import com.indeed.rabbitmq.admin.RabbitManagementApi;
+import com.indeed.rabbitmq.admin.pojo.OperatorPolicy;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class OperatorPolicyReconciler {
+
+    private static final Logger log = LoggerFactory.getLogger(PolicyReconciler.class);
+
+    private final RabbitManagementApiProvider apiProvider;
+
+    public OperatorPolicyReconciler(
```

Comment:
nit: do you want to one-line this?  (or do we have a code convention of writing constructors like this due to potentially long function signatures?)

Reply:
meh, i pretty much always do this format since more arguments often end up getting added. if you really hate it i can one-line it.

Reply:
Nope, no objections